app/components/AppHeader.tsx

```diff
@@ -81,19 +80,18 @@ interface AppHeaderProps {
   userLinks: { link: string; label: string }[];
 }
 
+const activeLinkStyle = { backgroundColor: "#E6EBF0", color: "#333333" };
+
 export function AppHeader({ links, userLinks }: AppHeaderProps) {
   const [opened, { toggle }] = useDisclosure(false);
-  const [active, setActive] = useState(links[0].link);
   const { classes, cx } = useStyles();
 
   const items = links.map((link) => (
     <NavLink
       key={link.label}
       to={link.link}
-      className={cx(classes.link, { [classes.linkActive]: active === link.link })}
-      onClick={() => {
-        setActive(link.link);
-      }}
+      className={({ isActive }) => cx(classes.link, { [classes.linkActive]: isActive })}
+      style={({ isActive }) => (isActive ? activeLinkStyle : {})}
```

Comment:
Probably don't need both className and style here? This appears to be applying the same styling twice.

Reply:
ugh you're right, my brain today lol

Reply:
the problem I came across is that I think those mantine styles can't be used in style